package.json

```diff
@@ -16,14 +15,28 @@
     "start": "electron . --debug --log-debug --log-to-console --disable-http-cache --translation-watch",
     "pack": "npm run build && electron-builder --dir",
     "dist": "npx electron-builder --mac dmg --win nsis portable --linux AppImage deb",
-    "build": "npm run build-main && npm run builder",
-    "build-main": "tsc --build src/main",
-    "builder": "node ./bin/build/builder.js",
-    "builder-help": "npm run builder -- --help",
-    "build-scss": "npm run builder -- --styles",
-    "build4production": "npm run build-main && npm run builder -- --prod",
-    "translations-pull": "tx pull -l en && tx pull -f",
-    "translations-update": "npm run translations-pull && npm run builder -- --translations",
+    "build": "npm run build:shared && npm run build:backend && npm run build:frontend",
+    "build:shared": "npm run build:shared:translations",
+    "build:shared:translations": "node bin/convert-translations-from-xml-to-json.js",
+    "build:backend": "tsc --build src/main",
+    "build:frontend": "npm run build:frontend:ts && npm run build:frontend:scss && npm run build:frontend:static",
+    "build:frontend:ts": "npm run build:frontend:ts:tsc && npm run build:frontend:ts:parcel",
+    "build:frontend:ts:tsc": "npx tsc -b src/renderer --pretty",
+    "build:frontend:ts:parcel": "npx parcel build tsc-dist/renderer/main.js --out-dir html-dist --out-file bundle.js --public-url ./ --target browser",
+    "build:frontend:scss": "npx node-sass scss/manifest.scss html-dist/main.css",
+    "build:frontend:static": "node ./bin/copy.js ./static ./html-dist",
+    "watch": "npm run watch:shared & npm run watch:frontend & npm run watch:backend",
+    "watch:shared": "npm run build:shared -- -w",
+    "watch:backend": "npm run build:backend -- -w",
+    "watch:frontend": "npm run watch:frontend:ts & npm run watch:frontend:scss && npm run watch:frontend:static",
+    "watch:frontend:ts": "npm run watch:frontend:ts:tsc & npm run watch:frontend:ts:parcel",
+    "watch:frontend:ts:tsc": "npm run build:frontend:ts:tsc -- -w --preserveWatchOutput",
+    "watch:frontend:ts:parcel": "npx parcel watch tsc-dist/renderer/main.js --out-dir html-dist --out-file bundle.js --public-url ./ --target browser",
+    "watch:frontend:scss": "npm run build:frontend:scss -- -w",
+    "watch:frontend:static": "npm run build:frontend:static -- -w",
+    "build4production": "NODE_ENV=production npm run build",
```

Comment:
this should also enable code minification

Reply:
But isn't this done with the NODE_ENV variable? Don't see any other step in your script.

Reply:
https://github.com/deltachat/deltachat-desktop/blob/master/bin/build/build-js.js#L28-L31